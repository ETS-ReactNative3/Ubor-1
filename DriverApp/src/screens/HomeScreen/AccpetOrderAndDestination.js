import React from "react";
import { View, Image, Text,Pressable } from "react-native";

const orderHandling =  (newOrder) => {

    <View style={{ alignItems: 'center' }}>
    <View style={{flexDirection: 'row', alignItems: 'center'}}>
    <View style={{ backgroundColor: '#d41212', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
    </View>
    <Text> km</Text>
    </View>
    </View>
    var order = {
        id: '1',
        type: 'UberX',
        originLatitude: 37.42342342342342,
        oreiginLongitude: -122.09495287867832,
        destLatitude: 28.452927,
        destLongitude: -16.260845,
        accept:true
    } 
    const dest = ["28.452927", "-16.260845"];
    if(newOrder.checker && newOrder.test == 'long') return "-16.260845";

    if(newOrder.checker) return "28.452927";

    if(newOrder.accept == true) {
        return "accepted order";
    } else {
        return "order declined";
    }
  }

export default orderHandling;
