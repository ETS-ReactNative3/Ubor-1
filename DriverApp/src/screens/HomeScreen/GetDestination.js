import React from "react";
import { View, Image, Text,Pressable } from "react-native";


import React from "react";
import { View, Image, Text,Pressable } from "react-native";

const getDestination =  (newOrder) => {

    <View style={{ alignItems: 'center' }}>
    <View style={{flexDirection: 'row', alignItems: 'center'}}>
    <View style={{ backgroundColor: '#d41212', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
    </View>
    <Text> km</Text>
    </View>
    </View>
    var order = {
        id: '1',
        type: 'UberX',
        originLatitude: 37.42342342342342,
        oreiginLongitude: -122.09495287867832,
        destLatitude: 28.452927,
        destLongitude: -16.260845,
        accept:true
    } 
    if (order && order.pickedUp) {
        return {
          latitude: order.latitude_destination,
          longitude: order.longitude_destination,
        }
      }
      return {
        latitude: order.latitude_rider,
        longitude: order.longitude_rider,
      }
  }

export default onAccept;
