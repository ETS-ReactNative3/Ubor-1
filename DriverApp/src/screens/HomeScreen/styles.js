import { StyleSheet, Dimensions } from 'react-native';


  /** 
 * This file holds the styling information index.js uses
*/
const styles = StyleSheet.create({
  bottomContainer: {
    height: 100,
    backgroundColor: 'white',
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
    padding: 20,
  },
  bottomText: {
    fontSize: 20,
    color: '#4a4a4a',
  },
  roundButton: {
    position: 'absolute',
    backgroundColor: 'orange',
    padding: 10,
    borderRadius: 20,
  },
  goButton: {
    position: 'absolute',
    backgroundColor: 'orange',
    width: 75,
    height: 75,
    justifyContent: 'center',
    alignItems: 'center',
    borderRadius: 60,
    bottom: 110,
    left: Dimensions.get('window').width / 2 - 37,
  },
  goText: {
    fontSize: 30,
    color: 'white',
    fontWeight: 'bold',
  },

  balanceButton: {
    position: 'absolute',
    backgroundColor: '#1c1c1c',
    width: 100,
    height: 50,
    justifyContent: 'center',
    alignItems: 'center',
    borderRadius: 40,
    top: 20,
    left: Dimensions.get('window').width / 2 - 50,
  },
  balanceText: {
    fontSize: 24,
    color: 'white',
    fontWeight: 'bold',
  }
});

export default styles;
