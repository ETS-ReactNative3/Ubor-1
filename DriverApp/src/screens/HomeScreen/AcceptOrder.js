import React, { useState, useEffect } from "react";
import { View, Text, Dimensions, Pressable } from "react-native";

import styles from './styles.js'

/** 
 * This component is responsible for the order acceptance button on the driver side
 * Contains function to accept and decline orders

 * The styles are contains within styles.js in this directory.
*/

const HomeScreen = () => {
  const [car,setCar] = useState(null);
  const [order, setOrder] = useState(null)
  const [newOrders, setNewOrders] = useState([{
    id: '1',
    type: 'UberX',

    originLatitude: 37.42342342342342,
    oreiginLongitude: -122.09495287867832,

    destLatitude: 28.452927,
    destLongitude: -16.260845,

    user: {
      rating: 4.8,
      name: 'Ciara',
    }
  },
  {
    id: '2',
    type: 'Uber Yello',

    originLatitude: 37.42342342342342,
    oreiginLongitude: -122.09495287867832,

    destLatitude: 28.452927,
    destLongitude: -16.260845,

    user: {
      rating: 4.8,
      name: 'Rahat',
    }
  }])


  const onDecline = () => {
    setNewOrders(newOrders.slice(1));
  }

  const onAccept =  (newOrder) => {
    setOrder(newOrder);
    setNewOrders(newOrders.slice(1));
    return 
  }

  const onDirectionFound = (event) => {
    console.log("Direction found: ");
    if (order) {
      setOrder({
        ...order,
        distance: event.distance,
        duration: event.duration,
        pickedUp: order.pickedUp || event.distance < 0.2,
        isFinished: order.pickedUp && event.distance < 0.2,
      })
    }
  }

  const getDestination = () => {
    if (order && order.pickedUp) {
      return {
        latitude: order.latitude_destination,
        longitude: order.longitude_destination,
      }
    }
    return {
      latitude: order.latitude_rider,
      longitude: order.longitude_rider,
    }
  }

  const renderBottomTitle = () => {
    if (order && order.isFinished) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center',justifyContent: 'center', backgroundColor: '#cb1a1a', width: 200, padding: 10,  }}>
            <Text style={{color: 'white', fontWeight: 'bold'}}>COMPLETE {order.type}</Text>
          </View>
          <Text style={styles.bottomText}>{order.Name}</Text>
        </View>
      )
    }

    if (order && order.pickedUp) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center'}}>
            <Text>{order.duration ? order.duration.toFixed(1) : '?'} min</Text>
            <View style={{ backgroundColor: '#d41212', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
            </View>
            <Text>{order.distance ? order.distance.toFixed(1) : '?'} km</Text>
          </View>
          <Text style={styles.bottomText}>Dropping off {order.Name}</Text>
        </View>
      )
    }

    if (order) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center'}}>
            <Text>{order.duration ? order.duration.toFixed(1) : '?'} min</Text>
            <View style={{ backgroundColor: '#1e9203', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
            </View>
            <Text>{order.distance ? order.distance.toFixed(1) : '?'} km</Text>
          </View>
          <Text style={styles.bottomText}>Picking up {order.Name}</Text>
        </View>
      )
    }
    if (car?.IsAvailable) {
      return (
        <Text style={styles.bottomText}>You're online</Text>
      )
    }
    return (<Text style={styles.bottomText}>You're offline</Text>);
  }

  return (
    <View>
      <Pressable
        onPress={() => console.warn('Balance')}
        style={styles.balanceButton}>
        <Text style={styles.balanceText}>
          <Text style={{ color: 'green' }}>$</Text>
          {' '}
          0.00
        </Text>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {top: 10, left: 10}]}>
        <Entypo name={"menu"} size={24} color="#4a4a4a"/>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {top: 10, right: 10}]}>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {bottom: 110, left: 10}]}>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {bottom: 110, right: 10}]}>
      </Pressable>

      <Pressable
        onPress={onGoPress}
        style={styles.goButton}>
        <Text style={styles.goText}>
          {car?.IsAvailable ? 'END' : 'GO'}
        </Text>
      </Pressable>

      <View style={styles.bottomContainer}>
        <Ionicons name={"options"} size={30} color="#4a4a4a"/>
          {renderBottomTitle()}
      </View>

    </View>
  );
};

export default HomeScreen;
