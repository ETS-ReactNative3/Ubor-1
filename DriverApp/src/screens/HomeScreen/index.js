import React, { useState, useEffect } from "react";
import { View, Text, Dimensions, Pressable } from "react-native";
import MapView, { PROVIDER_GOOGLE } from "react-native-maps";
import MapViewDirections from 'react-native-maps-directions';
import styles from './styles.js'
import Entypo from "react-native-vector-icons/Entypo";
import Ionicons from "react-native-vector-icons/Ionicons";
import NewOrderPopup from "../../components/NewOrderPopup";
import FontAwesome from "react-native-vector-icons/FontAwesome";
const GOOGLE_MAPS_APIKEY = 'AIzaSyA5VWIifqHFf9PuEpxXUUr72KgJa7lbQek';
import {API,graphqlOperation} from 'aws-amplify';
import { listDrivers } from "../../graphql/queries";
import { updateDriver } from "../../graphql/mutations.js";
import { listCustomerLocationDestinations } from "../../graphql/queries";


const HomeScreen = () => {
  const [car,setCar] = useState(null);
  const [myPosition, setMyPosition] = useState(null);
  const [order, setOrder] = useState(null)
  const [newOrders, setNewOrders] = useState([{
    id: '1',
    type: 'UberX',

    originLatitude: 37.42342342342342,
    oreiginLongitude: -122.09495287867832,

    destLatitude: 28.452927,
    destLongitude: -16.260845,

    user: {
      rating: 4.8,
      name: 'Ciara',
    }
  },
  {
    id: '2',
    type: 'Uber Yello',

    originLatitude: 37.42342342342342,
    oreiginLongitude: -122.09495287867832,

    destLatitude: 28.452927,
    destLongitude: -16.260845,

    user: {
      rating: 4.8,
      name: 'Rahat',
    }
  }])

  const fetchCar = async () =>{
    try{  
      const userData = await API.graphql(graphqlOperation(
        listDrivers,{
          filter: {License_no: {eq: 12}}
        }
      ))
      setCar(userData.data.listDrivers.items[0]);
    }catch(e){
      console.log(e);
    }
  }

  const fetchOrders = async () =>{
    try {
      const listAllOrders = await API.graphql(graphqlOperation(
        listCustomerLocationDestinations,{
          filter: {isAssigned: {eq: false}}
        }
      ))
      setNewOrders(listAllOrders.data.listCustomerLocationDestinations.items)
    } catch (error) {
      console.log(error)
    }
  }

  useEffect(()=>{
    fetchCar();
    fetchOrders();
  },[])

  const onGoPress = async () => {
    try {
      const updateCarState = await API.graphql(graphqlOperation(
        updateDriver,{
          input: {
            IsAvailable: !car.IsAvailable,
          id: car.id
          }
        }
      ))
      setCar(updateCarState.data.updateDriver)
    } catch (error) {
      console.log(error)
    }
  }
  const onDecline = () => {
    setNewOrders(newOrders.slice(1));
  }

  const onAccept =  (newOrder) => {
    console.log(newOrder)
    setOrder(newOrder);
    setNewOrders(newOrders.slice(1));
  }
  const onUserLocationChange = (event) => {
    setMyPosition(event.nativeEvent.coordinate);
  }

  const onDirectionFound = (event) => {
    console.log("Direction found: ");
    if (order) {
      setOrder({
        ...order,
        distance: event.distance,
        duration: event.duration,
        pickedUp: order.pickedUp || event.distance < 0.2,
        isFinished: order.pickedUp && event.distance < 0.2,
      })
    }
  }

  const getDestination = () => {
    if (order && order.pickedUp) {
      return {
        latitude: order.latitude_destination,
        longitude: order.longitude_destination,
      }
    }
    return {
      latitude: order.latitude_rider,
      longitude: order.longitude_rider,
    }
  }

  const renderBottomTitle = () => {
    if (order && order.isFinished) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center',justifyContent: 'center', backgroundColor: '#cb1a1a', width: 200, padding: 10,  }}>
            <Text style={{color: 'white', fontWeight: 'bold'}}>COMPLETE {order.type}</Text>
          </View>
          <Text style={styles.bottomText}>{order.Name}</Text>
        </View>
      )
    }

    if (order && order.pickedUp) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center'}}>
            <Text>{order.duration ? order.duration.toFixed(1) : '?'} min</Text>
            <View style={{ backgroundColor: '#d41212', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
              <FontAwesome name={"user"} color={"white"} size={20} />
            </View>
            <Text>{order.distance ? order.distance.toFixed(1) : '?'} km</Text>
          </View>
          <Text style={styles.bottomText}>Dropping off {order.Name}</Text>
        </View>
      )
    }

    if (order) {
      return (
        <View style={{ alignItems: 'center' }}>
          <View style={{flexDirection: 'row', alignItems: 'center'}}>
            <Text>{order.duration ? order.duration.toFixed(1) : '?'} min</Text>
            <View style={{ backgroundColor: '#1e9203', marginHorizontal: 10, width: 30, height: 30, alignItems:'center', justifyContent: 'center', borderRadius: 20}}>
              <FontAwesome name={"user"} color={"white"} size={20} />
            </View>
            <Text>{order.distance ? order.distance.toFixed(1) : '?'} km</Text>
          </View>
          <Text style={styles.bottomText}>Picking up {order.Name}</Text>
        </View>
      )
    }
    if (car?.IsAvailable) {
      return (
        <Text style={styles.bottomText}>You're online</Text>
      )
    }
    return (<Text style={styles.bottomText}>You're offline</Text>);
  }

  return (
    <View>
      <MapView
        style={{width: '100%', height: Dimensions.get('window').height - 120}}
        provider={PROVIDER_GOOGLE}
        showsUserLocation={true}
        onUserLocationChange={onUserLocationChange}
        initialRegion={{
          latitude: 37.42342342342342,
          longitude: -122.09495287867832,
          latitudeDelta: 0.02,
          longitudeDelta: 0.0121,
        }}
      >
        
        {order && (
          <MapViewDirections
            origin={myPosition}
            onReady={onDirectionFound}
            destination={getDestination()}
            apikey={GOOGLE_MAPS_APIKEY}
            strokeWidth={5}
            strokeColor="black"
          />
        )}
      </MapView>

      <Pressable
        onPress={() => console.warn('Balance')}
        style={styles.balanceButton}>
        <Text style={styles.balanceText}>
          <Text style={{ color: 'green' }}>$</Text>
          {' '}
          0.00
        </Text>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {top: 10, left: 10}]}>
        <Entypo name={"menu"} size={24} color="#4a4a4a"/>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {top: 10, right: 10}]}>
        <Entypo name={"menu"} size={24} color="#4a4a4a"/>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {bottom: 110, left: 10}]}>
        <Entypo name={"menu"} size={24} color="#4a4a4a"/>
      </Pressable>

      <Pressable
        onPress={() => console.warn('Hey')}
        style={[styles.roundButton, {bottom: 110, right: 10}]}>
        <Entypo name={"menu"} size={24} color="#4a4a4a"/>
      </Pressable>

      <Pressable
        onPress={onGoPress}
        style={styles.goButton}>
        <Text style={styles.goText}>
          {car?.IsAvailable ? 'END' : 'GO'}
        </Text>
      </Pressable>

      <View style={styles.bottomContainer}>
        <Ionicons name={"options"} size={30} color="#4a4a4a"/>
          {renderBottomTitle()}
        <Entypo name={"menu"} size={30} color="#4a4a4a" />
      </View>

      {newOrders.length > 0 && !order && <NewOrderPopup
        newOrder={newOrders[0]}
        duration={2}
        distance={0.5}
        onDecline={onDecline}
        onAccept={() => onAccept(newOrders[0])}
      />}
    </View>
  );
};

export default HomeScreen;
