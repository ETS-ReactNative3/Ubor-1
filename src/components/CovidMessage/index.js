import React from 'react';
import {View,Text} from 'react-native';
//import HomeMap from '../../components/HomeMap';
import styles from './styles';

const CovidMessage = (props) => {
    return (
        <View style={styles.container}>
           <Text style={styles.title}>
               Travel only if neccessary
           </Text>
           <Text style={styles.text}>
               Covid BS.
               Flatten the curve and shit.
           </Text>
           <Text style={styles.learnMore}>
               Learn More
           </Text>
        </View>
    );
}

export default CovidMessage