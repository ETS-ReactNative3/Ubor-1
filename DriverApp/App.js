/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

 import React from 'react';
 import {Auth} from 'aws-amplify'
import { withAuthenticator} from 'aws-amplify-react-native';
 
 import {
   SafeAreaView,
   StatusBar,
 } from 'react-native';
 import HomeScreen from './src/screens/HomeScreen';
 
 const App: () => React$Node = () => {
   return (
     <>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView>
         <HomeScreen />
       </SafeAreaView>
     </>
   );
 };

 const signUpConfig = {
  header: "My Customized Sign Up",
  hideAllDefaults: true,
  signUpFields: [
    {
      label: "Username",
      key: "username",
      required: true,
      displayOrder: 1,
      type: "string",
    },
    {
      label: "Password",
      key: "password",
      required: true,
      displayOrder: 2,
      type: "password",
    },
    {
      label: "License Plate Number",
      key: "custom:license",
      required: true,
      displayOrder: 3,
      type: "string",
    },
    {
      label: "Car Colour",
      key: "custom:carColour",
      required: true,
      displayOrder: 4,
      type: "number",
    },
    {
      label: "Email",
      key: "email",
      required: true,
      displayOrder: 5,
      type: "string",
    },
    {
      label: "First Name",
      key: "custom:fname",
      required: true,
      displayOrder: 6,
      type: "string",
    },
    {
      label: "Last Name",
      key: "custom:lname",
      required: true,
      displayOrder: 7,
      type: "string",
    },
    {
      label: "Phone Number",
      key: "phone_number",
      required: true,
      displayOrder: 8,
      type: "string",
    },
    {
      label: "Date of Birth",
      key: "custom:dob",
      required: true,
      displayOrder: 9,
      type: "string",
    }
  ],
};
 
export default withAuthenticator(App, {signUpConfig}) ;
 