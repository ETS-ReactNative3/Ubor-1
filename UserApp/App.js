/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React, { useEffect } from 'react';
import 'react-native-gesture-handler'
import {StatusBar, PermissionsAndroid, Platform} from 'react-native';
import Geolocation from '@react-native-community/geolocation';
import {Auth} from 'aws-amplify'
import { withAuthenticator} from 'aws-amplify-react-native';
 
import SearchResults from './src/screens/SearchResults';
import HomeScreen from './src/screens/HomeScreen';
import DestinationSearch from './src/screens/DestinationPage';
import Router from './src/navigation/Root'
import UberTypes from './src/components/UberTypes';

navigator.geolocation = require('@react-native-community/geolocation');

 
 const App: () => React$Node = () => {
  const androidPermission = async () => {
    try {
      const granted = await PermissionsAndroid.request(
        PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
        {
          title: "Uber App Location Permission",
          message:
            "Uber App needs access to your location " +
            "so you can take awesome rides.",
          buttonNeutral: "Ask Me Later",
          buttonNegative: "Cancel",
          buttonPositive: "OK"
        }
      );
      if (granted === PermissionsAndroid.RESULTS.GRANTED) {
        console.log("You can use the location");
      } else {
        console.log("Location permission denied");
      }
    } catch (err) {
      console.warn(err);
    }
  }

  useEffect(() => {
    if (Platform.OS === 'android') {
      androidPermission();
      // Auth.currentUserInfo().then(info => {console.log(info)});
    } else {
      // IOS
      Geolocation.requestAuthorization();
    }
  }, [])

   return (
     <>
       <StatusBar barStyle="dark-content" />
       <Router />
     </>
   );
 };
 const signUpConfig = {
  header: "My Customized Sign Up",
  hideAllDefaults: true,
  signUpFields: [
    {
      label: "Username",
      key: "username",
      required: true,
      displayOrder: 1,
      type: "string",
    },
    {
      label: "Password",
      key: "password",
      required: true,
      displayOrder: 2,
      type: "password",
    },
    {
      label: "Email",
      key: "email",
      required: true,
      displayOrder: 3,
      type: "string",
    },
    {
      label: "First Name",
      key: "custom:fname",
      required: true,
      displayOrder: 4,
      type: "string",
    },
    {
      label: "Last Name",
      key: "custom:lname",
      required: true,
      displayOrder: 5,
      type: "string",
    },
    {
      label: "Phone Number",
      key: "phone_number",
      required: true,
      displayOrder: 6,
      type: "string",
    },
    {
      label: "Date of Birth",
      key: "custom:dob",
      required: true,
      displayOrder: 7,
      type: "string",
    }
  ],
};
 
 export default withAuthenticator(App, {signUpConfig}) ; 