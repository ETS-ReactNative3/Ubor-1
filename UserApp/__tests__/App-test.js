/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';
import HomeScreen from '../src/screens/HomeScreen';
import CovidMessage from '../src/components/CovidMessage';
import { create } from 'react-test-renderer';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  expect(HomeScreen).toTruthy();
  // expect(tree).toMatchSnapshot();
});
