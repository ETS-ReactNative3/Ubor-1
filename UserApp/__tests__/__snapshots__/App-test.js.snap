// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#1065e9",
      "borderTopLeftRadius": 10,
      "borderTopRightRadius": 10,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 10,
      }
    }
  >
    Travel only if neccessary
  </Text>
  <Text
    style={
      Object {
        "color": "#bed9ff",
        "fontSize": 15,
        "marginBottom": 10,
      }
    }
  >
    Covid BS. Flatten the curve and shit.
  </Text>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 15,
        "fontWeight": "bold",
      }
    }
  >
    Learn More
  </Text>
</View>
`;
